src/components/elements/NavigationBar.jsx

```diff
@@ -1,108 +1,45 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { useAuth } from 'hooks/authentication';
+import useDarkMode from 'hooks/dark-mode';
```

Comment:
Use "kebab case for file naming and make method naming for camel case consistent.